packages/mui-joy/src/ListItemAdornment/ListItemAdornment.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { OverridableComponent } from '@mui/types';
+import composeClasses from '@mui/base/composeClasses';
+import { styled, useThemeProps } from '../styles';
+import { ListItemAdornmentProps, ListItemAdornmentTypeMap } from './ListItemAdornmentProps';
+import { getListItemAdornmentUtilityClass } from './listItemAdornmentClasses';
+
+const useUtilityClasses = (ownerState: ListItemAdornmentProps) => {
+  const slots = {
+    root: ['root', ownerState.end && 'end'],
+  };
+
+  return composeClasses(slots, getListItemAdornmentUtilityClass, {});
+};
+
+const ListItemAdornmentRoot = styled('span', {
```

Comment:
```suggestion
const ListItemDecorationRoot = styled('span', {
```

Just not too sure about the _adornment_ word,  seems a hard one. Thought of _decoration_ as a possible suggestion!

Reply:
updated to ListItemDecorator